packages/common/src/lib/form-dialog/form-dialog.component.html

```diff
@@ -0,0 +1,31 @@
+<h1 mat-dialog-title>{{ data.title | translate }}</h1>
+
+<igo-form
+  *ngIf="form$ | async as form"
+  [form]="form"
+  [formData]="data$ | async"
+  (submitForm)="onSubmit($event)"
+>
+  <igo-form-group [group]="form.groups[0]"></igo-form-group>
+
+  <div formButtons mat-dialog-actions>
+    <button mat-stroked-button type="button" color="primary" (click)="cancel()">
+      {{ data.cancelButtonText | translate }}
+    </button>
+    <button
+      mat-flat-button
+      type="submit"
+      color="primary"
+      [disabled]="isDisabled$ | async"
```

Comment:
Est-ce qu'on serait mieux de laisser le bouton cliquable et afficher une erreur lorsque le formulaire est non valide? On pourrait aussi focuser le premier control non valide ce qui permettrait d'aider l'usager a remplir le formulaire sans devoir chercher pourquoi le bouton n'est pas cliquable
```suggestion
      [disabled]="isDisabled$ | async"
```

Reply:
Ok pour le laisser clicable. bonne idée.

Par contre, je n'ai pas accès a des sélecteurs pour aller placer le focus dans le 1er en erreur.